pkg/chart/chart.go

```diff
@@ -323,24 +331,32 @@ func (c *HelmRepoClient) Init(appRepo *appRepov1.AppRepository, caCertSecret *co
 
 // GetChart retrieves and loads a Chart from a registry in both
 // v2 and v3 formats.
```

Comment:
Thanks for cleaning up the imports etc. The above comment is probably now untrue too.